eng/pipelines/scripts/inject_codesnippets.py

```diff
@@ -21,8 +21,8 @@
 HTML_ESCAPE_TABLE = {
     "&": "&amp;",
     '"': "&quot;",
-    ">": "&gt;",
-    "<": "&lt;",
+    ">": "&amp;gt;",
```

Comment:
Would the other entries need this double escaping as well?

Reply:
Nope. This ended up being a javadoc intricacy/bug. 

I'm surrounding with `@code` and NOT replacing it now.